crates/rolldown/tests/rolldown/function/module_types/asset/artifacts.snap

```diff
@@ -0,0 +1,28 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## assets/foo-vpuLoj-k.txt
+
+```txt
+hello, world
+```
+## main.js
+
+```js
+
+//#region foo.txt
+var foo_default = "TODO: should be file path";
```

Comment:
This would be fixed in later PRs. It requires some refactors. If I do it this PR, that would cause too many changes.